README.md

```diff
@@ -1 +1,66 @@
 # wizard-additional-playing-cards
+
+## Tornado | 14 | blau
+
+- Nach dem Stich wird in die andere Richtung gespielt
+- Bis zum Ende der Runde
+
+## Katapult | 14 | rot
+
+- Alle ziehen eine Karte
+
+## Frosch | 14 | grün
+
+- Alle müssen eine Karte auf den Stapel legen
+
+## Affe | 14 | gelb
+
+- Beim ausspielen wird die Farbkarte umgedreht
+
+## Nagel+Hammer | N | weiß
+
+- Der aktuelle Stich bleibt liegen und der nächste Sich wird darauf weiter gespielt
+- Der Gewinner bekommt 2 Stiche
+
+## Wage | 4/11 | bunt
+
+- Die Niedrigste Karte gewinnt
+- Bei zwei Narren gewinnt der 1. Narre
```

Comment:
Wenn eine Fee und ein Drache in dem Stich sind, gewinnt dann der Drache?

Reply:
Wenn man nur zu zweit spielt, dann ja.

Reply:
Wenn man zu dritt spielt, wer gewinnt dann?
Wir haben eine Fee, einen Drachen und einen Narren.
Gewinnt die Fee, der Drache oder der Narre?